ddspipe_core/src/cpp/core/DdsPipe.cpp

```diff
@@ -343,7 +347,16 @@ void DdsPipe::create_new_bridge_nts_(
 
     try
     {
-        auto new_bridge = std::make_unique<DdsBridge>(topic, participants_database_, payload_pool_, thread_pool_);
+        // Use topic specific forwarding routes if available
+        RoutesConfiguration routes_config = topic_routes_config_().count(topic) !=
```

Comment:
I think it would be more readable if the one-liner if statement was replaced by a normal if statement.